extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/PrivilegesBuilder.scala

```diff
@@ -93,6 +101,16 @@ object PrivilegesBuilder {
       }
     }
 
+    def mergeProjectionWithIdentifier(table: Identifier, plan: LogicalPlan): Unit = {
```

Comment:
how about projectV2Table and projectV1Table

Reply:
good. much clearer.